hammer/library/jiraoperations.py

```diff
@@ -14,23 +14,60 @@
     'ticket_assignee_id'
     ])
 
+class JiraLabels(object):
+    """ Base class for JIRA tickets labeling """
+    DEFAULT_LABELS = {
+        'cloudtrails': 'cloudtrail-issue',
```

Comment:
Doesn't jira expect labels as a list? I see you have lists in configuration file.